### DISCUSSION TITLE
Build App Windows 10

### DISCUSSION THREAD

**Discussion by mingle47** [nem0/LumixEngine#1379] [General]
I loaded the demo universe and then tried to build the app. I received two errors.

"Loading settings...
Initializing imgui...
Loading settings...
Startup took 0.619642 s
Loading universe demo...
Parsing universe...
Universe parsed in 0.136307 seconds
Failed to copy D:\lumixOpenGL\dbghelp.dll to E:\Lumix\build/dbghelp.dll
Failed to copy D:\lumixOpenGL\dbgcore.dll to E:\Lumix\build/dbgcore.dll
Exporting finished."

The two dlls are not in the source directory.
Could not get the app to run - looked like a skybox only being rendered.

**Comment by nem0** [comment]
Hi, DLLs are here https://github.com/nem0/LumixEngine/blob/master/external/dbghelp/dbghelp.dll, but you can ignore that warning. It does not cause any issues, other than worse callstacks in case of crash. I will fix this so it does not print errors. 

By default, `main` universe is run in exported game, but there's none unless you create it, so it displays empty scene == just skybox. #1380

**Reply by mingle47** [reply]
Thanks for prompt reply

**Reply by nem0** [reply]
Both issues should be fixed now
